### PR TITLE
adding php 7.4 runtime image in oryx

### PR DESCRIPTION
**Pull request by arroyc**
1. building php 7.4 apache images in buildpack-deps:stretch
2. added runtime image teest for the corresponding images

next PR
1. support for php 7.4 in build image 
2. integration tests

### CODE REVIEW

File: images/runtime/php/template.base.Dockerfile [hunk 1]
```diff
@@ -25,12 +25,25 @@ RUN apt-get update \
...
-    && docker-php-ext-configure gd --with-png-dir=/usr --with-jpeg-dir=/usr \
-    && docker-php-ext-configure imap --with-kerberos --with-imap-ssl \
-    && docker-php-ext-configure pdo_odbc --with-pdo-odbc=unixODBC,/usr \
-    && docker-php-ext-install gd \
-        mysqli \
+    && ln -s /usr/include/x86_64-linux-gnu/gmp.h /usr/include/gmp.h
+
+RUN set -eux; \
+    if [[ $PHP_VERSION == 7.4.* ]]; then \
+		echo "hellow: "$PHP_VERSION \
```
Comment (arroyc) [inline images/runtime/php/template.base.Dockerfile]: need to remove this

File: images/runtime/php/7.4/7.4.Dockerfile [hunk 2]
```diff
@@ -0,0 +1,324 @@
+# From https://github.com/docker-library/php.git
+FROM php-run-base
+ARG IMAGES_DIR=/tmp/oryx/images
+
+# prevent Debian's PHP packages from being installed
+# https://github.com/docker-library/php/pull/542
+#RUN set -eux; \
```
Comment (arroyc) [inline images/runtime/php/7.4/7.4.Dockerfile]: need to remove all these commeneted codes from line 5 to 56

File: images/runtime/php/runbase.Dockerfile [hunk 3]
```diff
@@ -46,10 +47,19 @@ RUN set -eux; \
 		unixodbc-dev \
 		openssh-server \
 		vim \
-		curl \
 		wget \
 		tcptraceroute \
 		mariadb-client \
 		openssl \
+        libedit-dev \
```
Comment (arroyc) [inline images/runtime/php/runbase.Dockerfile]: spacing

File: images/runtime/php/template.base.Dockerfile [hunk 4]
```diff
@@ -55,7 +68,7 @@ RUN apt-get update \
         sysvsem \
         sysvshm \
         pdo_odbc \
-        wddx \
+#        wddx \
```
Comment (arroyc) [inline images/runtime/php/template.base.Dockerfile]: wddx is deprecated in 7.4

File: images/runtime/php/template.base.Dockerfile [hunk 5]
```diff
@@ -25,12 +25,25 @@ RUN apt-get update \
...
+        && PHP_OPENSSL=yes docker-php-ext-configure imap --with-kerberos --with-imap-ssl ; \
+    else \
+		echo "it's not php 7.4" \
+		&& echo $PHP_VERSION \
+        && docker-php-ext-configure gd --with-png-dir=/usr --with-jpeg-dir=/usr \
+        && docker-php-ext-configure imap --with-kerberos --with-imap-ssl ; \
+    fi
+
+RUN docker-php-ext-configure pdo_odbc --with-pdo-odbc=unixODBC,/usr \
+    && docker-php-ext-install mysqli \
```
Comment (arroyc) [inline images/runtime/php/template.base.Dockerfile]: need to install gd here

File: tests/Oryx.RuntimeImage.Tests/Php/PhpImageTest.cs [hunk 6]
```diff
@@ -80,13 +82,15 @@ public void GraphicsExtension_Gd_IsInstalled(string imageTag)
 
             // Assert
             JObject gdInfo = JsonConvert.DeserializeObject<JObject>(result.StdOut);
+            //Assert.Contains((((JValue)gdInfo.GetValue("GIF Read Support")).Value).ToString(), "true");
```
Comment (dorfire) [inline tests/Oryx.RuntimeImage.Tests/Php/PhpImageTest.cs]: why commented out?

**Pull request merged**